pages/accountLists/[accountListId]/settings/preferences.page.tsx

```diff
@@ -61,7 +61,9 @@ const Preferences: React.FC = () => {
 
   useEffect(() => {
     const redirectToDownloadExportedData = (exportDataExportId: string) => {
-      if (!exportDataExportId) {return;}
+      if (!exportDataExportId) {
```

Comment:
Does this check do anything? If `query.exportId` is an empty string, we aren't calling `redirectToDownloadExportedData`.